src/weather_model_graphs/save.py

```diff
@@ -59,6 +61,13 @@ def to_pyg(
             "install weather-mode-graphs[pytorch] to enable writing to torch files"
         )
 
+    # Default values for arguments
+    if edge_features is None:
+        edge_features = ["len", "vdiff"]
```

Comment:
This makes sense as a default as it matches neural-lam (https://github.com/mllam/neural-lam/blob/a2ddcd4298d7865225c45866e7495664b3e382ab/neural_lam/create_mesh.py#L84) :+1: